backend/controllers/prisonerProfile/prisonerFinances/prisonerDamageObligations.js

```diff
@@ -58,6 +58,7 @@ module.exports = ({ prisonApi }) => async (req, res) => {
         offenderNo,
       },
       rows,
+      showDamageObligationsLink: true,
       totalOwed: formatCurrency(totalOwed),
     })
   } catch (error) {
```

Comment:
You should be able to use the new error handler. If the redirect url is the original one you can just let the error bubble up. If not you need to set res.locals.redcirectUrl and then re-throw the error.

Reply:
Nice one, will have a look